server/src/main/java/com/workshop/domain/Personel.java

```diff
@@ -0,0 +1,29 @@
+package com.workshop.domain;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Table(name = "PERSONEL")
+public class Personel {
```

Comment:
Shouldn't it be embedded in User class, so that we don't have separate entities describing users and workers?

Reply:
Yes, we should merge those two classes